bom/compile/pom.xml

```diff
@@ -187,12 +187,24 @@
       <artifactId>javaxcomm</artifactId>
       <version>1.0.1</version>
       <scope>compile</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-net</groupId>
+          <artifactId>commons-net</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>com.neuronrobotics</groupId>
       <artifactId>nrjavaserial</artifactId>
-      <version>3.12.0</version>
+      <version>3.14.0</version>
```

Comment:
Why not nrjavaserial `5.2.1` ?

Reply:
There is a difference between compile and runtime dependencies. The compile time dependencies are used by bnd to compute version ranges for package dependencies. So you want these computed ranges to be as big as possible so you do not unnecessarily force users of the framework to upgrade their bundles.